src/app/sharedUI/molecules/ImagePlaceholder.tsx

```diff
@@ -0,0 +1,11 @@
+import { ArtsyMarkIcon, useColor, Flex } from "palette"
+
+export const ImagePlaceholder = ({ height = 100 }: { height: number }) => {
+  const color = useColor()
+
+  return (
+    <Flex alignItems="center" justifyContent="center" height={height} background={color("black10")}>
```

Comment:
```suggestion
    <Flex alignItems="center" justifyContent="center" height={height} backgroundColor={"black10"}>
```
You dont need `useColor()`

Reply:
remove the `{` `}` to make it even nicer :)

Reply:
it would be great if we can add this stylistic thing to our precommit lint hook or prettier on autosave